OpenTrons/opentrons-api
# fix(protocol-designer): switch inputfield from old one to new one

# Overview
switch`InputField` from old one to new one.
utilize Styled Textarea instead of creating a new component.

close RQA-3371 and RQA-3383




## Test Plan and Hands on Testing
1. create a new protocol
2. fill out the form(name, description, and organization/author)
check the above items area displayed in Protocol Overview page


## Changelog



## Review requests



## Risk assessment
low
